pkg/operator/csidriveroperator/csioperatorclient/powervs-block.go

```diff
@@ -0,0 +1,38 @@
+package csioperatorclient
+
+import (
+	"os"
+	"strings"
+
+	configv1 "github.com/openshift/api/config/v1"
+)
+
+const (
+	POWERVSBlockCSIDriverName          = "powervs.csi.ibm.com"
+	envPOWERVSBlockDriverOperatorImage = "POWERVS_BLOCK_CSI_DRIVER_OPERATOR_IMAGE"
+	envPOWERVSBlockDriverImage         = "POWERVS_BLOCK_CSI_DRIVER_IMAGE"
```

Comment:
Should it be `*PowerVS*` everywhere (incl. method names and condition prefix below)? Or is POWER an abbreviation?

Reply:
Yes @jsafrane , PowerVS represents IBM Power Systems Virtual Servers.
Reference: https://www.ibm.com/in-en/products/power-virtual-server

Reply:
I prefer to keep `PowerVS`, `POWER` is too generic.

Reply:
and POWER isn't an abbreviation

Reply:
The variable names changed from `POWERVS` to `PowerVS`.